### PR TITLE
Adopt main as default branch name

### PR DESCRIPTION
**Pull request by xla**
[Rendered](https://github.com/radicle-dev/radicle-decisions/blob/xla/main-branch-proposal/proposals/0003.md)

### CODE REVIEW

**Comment by sarahscott** [comment]
👏

**Comment by cloudhead** [comment]
`trunk` is very radicle, but then again it's the bitcoin `main` chain.

**Review by kim — CHANGES_REQUESTED** [review]
It is impossible to have a reasonable conversation about a proposal like this. Which doesn’t belong here.

**Comment by MeBrei** [comment]
I think this is a discussion worth having, especially as we are ourselves in the process of shaping a new code collaboration platform. This is a topic that has come up many times, so will most likely as well when radicle goes live.

I have also seen `root` or `primary` being used as alternatives.

**Comment by buzdyk** [comment]
> inclusive environment

I feel like a brute by even asking this but who feels offended by the `master` branch name?
Anyone?

**Review by FintanH — APPROVED** [review]
I suppose I'll give my 2c on this.

Initially, I thought: "How does this relate to "master/slave" connotations other than it being the word "master"?" My thinking was that the meaning was actually to do with a master copy, i.e. like in the music industry they would make a master tape and make copies of it for distribution.

Now, I don't know where this terminology originated from. I was told by a peer of mine that an archivist/historian found that it did have negative connotations.

So, in summary, I think that if there's even a _chance_ this might offend someone, then this is something that's within our ability to change.

What I don't see here is a plan to actually enact these changes. The repositories, the related code, how can people migrate to this.

Finally, I think `radicle` could be a good default name for our branches.

**Comment by xla** [comment]
> So, in summary, I think that if there's even a _chance_ this might offend someone, then this is something that's within our ability to change.

This is my personal motiviation as well.

> What I don't see here is a plan to actually enact these changes. The repositories, the related code, how can people migrate to this.

What exactly are you looking for, it's configuration change in each repo settings. For the use in the application it would require a PR addressing all occurrences.

> Finally, I think `radicle` could be a good default name for our branches.

Doesn't really carry any meaning, is it just to make it clear it's radicle managed projects?

**Comment by FintanH** [comment]
> What exactly are you looking for, it's configuration change in each repo settings. For the use in the application it would require a PR addressing all occurrences.

Well, these decisions usually come with a plan of how to change things as well as the motivation. So like an audit of where we need to change `master` would be useful. Which codebases are we targeting for these changes? Maybe it's as easy as just changing the settings, but I feel like there's more to it than that.

> Doesn't really carry any meaning, is it just to make it clear it's radicle managed projects?

Ya, I mean the name is pretty arbitrary as long as it's not offensive. I think `radicle` is cool because they're `radicle` projects.

File: proposals/0003.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+* it will require some administration on our code hosting platform
+* local checkouts have to be adjusted
+
+# Open Questions
+
+* Can we think of potential alternatives to `main` (e.g. `trunk`)?
+
+# Future possibilities
+
+* A more safe environment for more
```
Comment (sarahscott) [inline proposals/0003.md:38]: more what?

**Comment by sarahscott** [comment]
I agree with @MeBrei 's point; this issue isn't going anywhere and if we have the opportunity to move our industry away from unfortunate and embarrassing metaphors, we should.
